package.json

```diff
@@ -5,6 +5,10 @@
     "test-api": "cypress run --headless --spec 'cypress/integration/api_tests/*'"
   },
   "devDependencies": {
-    "cypress": "^7.3.0"
+    "cypress": "^7.5.0",
+    "cypress-otp": "^1.0.3"
+  },
+  "dependencies": {
+    "twilio": "^3.63.0"
```

Comment:
I think we can remove this as we are not actually using them right?

Reply:
Already removed the twilio and cypress-otp dependencies